packages/remix-server-runtime/interface/responses.ts

```diff
@@ -0,0 +1,2 @@
+export type Json = <Data>(data: Data, init?: number | ResponseInit) => Response
```

Comment:
Likewise, I probably would've preferred an `interface` here, like:

```ts
export interface JsonFunction<Data> {
  (data: Data, init?: number | ResponseInit): Response;
}
```

I think we should also name our interfaces like `XFunction` to indicate they are functions. How do you feel about that?

Reply:
I'm fine with that, especially since its an already-established convention in Remix codebase. 👍